src/models/Body.ts

```diff
@@ -0,0 +1,32 @@
+import { ResponseData } from "../networking/NetworkResponse";
```

Comment:
Going to skip over all of these for now, they all look fine, it would be good to eventually move these to be auto-generated files.

Maybe a project would be to split out the database models from cdp-backend into it's own repo as JSON files and have `cdp-backend` and `cdp-frontend` auto-generate these types of model classes.

Again this is way way down the line.

Reply:
That would indeed be nice

Reply:
Agree with Jackson, to avoid manually keeping these files and database models in sync.

Reply:
@JacksonMaxfield Can you make an issue for this - it doesn't really belong on cdp-frontend's board solo.  I'm happy to do the work once we have either a simple flatfile or separate repo. 
My plan would be to have a node script that writes new model files based on the schema provided by the other repo or flatfile.

Reply:
Yep. I will add one tonight!

Reply:
Created and tracking here: https://github.com/CouncilDataProject/cdp-roadmap/issues/13